src/test/java/lotto/domain/ticket/TicketsTest.java

```diff
@@ -0,0 +1,51 @@
+package lotto.domain.ticket;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import lotto.AppConfig;
+import lotto.domain.rank.Rank;
+import lotto.domain.ticket.generator.CustomTicketGenerator;
+import lotto.dto.TicketDto;
+
+class TicketsTest {
+
+    private static final AppConfig APP_CONFIG = AppConfig.getInstance();
+
+    private final CustomTicketGenerator customTicketGenerator = APP_CONFIG.ticketGenerator;
+
+    private Tickets generateTickets(final List<TicketDto> generatedTickets, final int ticketCount) {
+        customTicketGenerator.initTickets(generatedTickets);
+        return new Tickets(ticketCount, customTicketGenerator);
+    }
+
+    @DisplayName("로또 목록 생성, 개수 일치 확인 테스트")
+    @ParameterizedTest(name = "[{index}] 로또 개수 : {1}")
+    @MethodSource("lotto.domain.ticket.provider.TicketsTestProvider#provideForGenerateTest")
+    void generateTicketsSizeCheckTest(final List<TicketDto> generatedTickets, final int ticketCount) {
+        final Tickets tickets = generateTickets(generatedTickets, ticketCount);
+        assertThat(tickets.getTickets().size()).isEqualTo(ticketCount);
```

Comment:
```suggestion
        assertThat(tickets.getSize()).isEqualTo(ticketCount);
```

이미 직접 리턴하는 메서드가 있네요.

Reply:
그렇군요!! 날카우십니다ㅎ 수정했습니다!!!